pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -310,6 +342,10 @@ func UpsertIPsecEndpoint(local, remote *net.IPNet, dir IPSecDir) (uint8, error)
 
 // DeleteIPsecEndpoint deletes a endpoint associated with the remote IP address
 func DeleteIPsecEndpoint(remote *net.IPNet) {
+	scopedLog := log.WithFields(logrus.Fields{
+		"remote-ip": remote.IP,
+	})
+	scopedLog.Info("Delete IPsec Endpoint")
```

Comment:
How often will this occur? We're trying to get to a point where there is little to no `Info()` messages after startup. Maybe it's okay if this is likely to just be once per node add/remove event. (/cc @tgraf )

Reply:
Good catch lets just remove it. Its not so useful except when trying to find delete events but its better to debug the nodeDelete and k8s watcher at that point.